hkmc2Benchmarks/src/test/bench/LazySpreads.mls

```diff
@@ -343,11 +336,10 @@ suite.add("EagerBuiltSillyEager15", () =>
   sillyEager(xs))
 
 :silent
-let configs = [
-  { name: "LazyArray", lazySlice: LazyArray.slice, lazyConcat: LazyArray.__concat, split: LazyArray.__split },
-  { name: "FingerTreeList", lazySlice: FingerTreeList.slice, lazyConcat: FingerTreeList.__markerConcat, split: FingerTreeList.__split },
-  { name: "LazyFingerTree", lazySlice: LazyFingerTree.slice, lazyConcat: LazyFingerTree.__concat, split: LazyFingerTree.__split }
-]
+let configs = Array of
```

Comment:
The `Array` function is a terrible footgun.

```js
Welcome to Node.js v22.11.0.
Type ".help" for more information.
> Array()
[]
> Array(1)
[ <1 empty item> ]
> Array(1, 2)
[ 1, 2 ]
```

I meant to write `tuple` not `array` in my suggestion abioe.